src/grid/cell.component.ts

```diff
@@ -68,7 +68,7 @@ export class IgxGridCellComponent implements IGridBus, OnInit {
         return this.gridAPI.get(this.gridID);
     }
 
-    get rowIndex(): number {
+    get rowIndex(): any {
```

Comment:
Why did you change the return type? Index should still be a number.

Reply:
The return type was changed with the initial implementation idea, it was an oversight not to revert it, thanks for pointing it out!